modules/auction-manager/src/lib.rs

```diff
@@ -82,42 +86,49 @@ pub struct CollateralAuctionItem<AccountId, BlockNumber> {
 	/// Current collateral amount for sale
 	#[codec(compact)]
 	amount: Balance,
-	/// Target sales amount of this auction
+	/// Base sales amount of this auction
 	/// if zero, collateral auction will never be reverse stage,
 	/// otherwise, target amount is the actual payment amount of active
 	/// bidder
 	#[codec(compact)]
-	target: Balance,
+	base: Balance,
+	/// Penalty amount as part of the auction.
+	#[codec(compact)]
```

Comment:
We shouldn't really mark those compact if they are not part of extrinsic parameters. Given we need a migration, now is good time to remove all the compact here.